packages/mobile/patch/rnfl/AndroidManifest.xml.patch

```diff
@@ -0,0 +1,17 @@
+--- node_modules/react-native-file-logger/android/src/main/AndroidManifest.xml	2025-03-11 23:02:55
```

Comment:
These two patches implement functionality that it's in review on `react-native-file-logger` but haven't been released yet.